apps/api/src/applicant/entity/ienapplicant-status.entity.ts

```diff
@@ -18,6 +21,10 @@ export class IENApplicantStatus {
   @Exclude()
   full_name?: string;
 
+  @Column({type:'varchar', length:256,nullable:true})
+  category?:string
```

Comment:
Does `category` replace `parent`?

Reply:
Thats the plan. I didn't have a formal structure for the data, but the category field mapped very nicely onto the status category enum we already had.